base_layer/wallet/src/storage/sqlite_db.rs

```diff
@@ -187,6 +192,26 @@ impl WalletSqliteDatabase {
         Ok(())
     }
 
+    fn set_node_id(&self, node_id: &NodeIdentity, conn: &SqliteConnection) -> Result<(), WalletStorageError> {
+        let cipher = acquire_read_lock!(self.cipher);
+        WalletSettingSql::new(DbKey::CommsPublicKey.to_string(), node_id.public_key().to_hex()).set(&conn)?;
+        WalletSettingSql::new(DbKey::CommsAddress.to_string(), node_id.public_address().to_string()).set(&conn)?;
+        WalletSettingSql::new(DbKey::CommsFeatures.to_string(), node_id.features().bits().to_string()).set(&conn)?;
```

Comment:
Why are features saved to the DB? Are these configurable?

Reply:
features is, 
The other two is purely convenance so that we dont have to recalc them. I added everything from the node_id.json file.